src/components/WwwFrame/TheHeader.vue

```diff
@@ -597,6 +625,31 @@ export default {
 			}
 		}
 	},
+	created() {
+		// MARS-194 User Metrics for Optimizely A/B experiment
+		const hasEverLoggedInBefore = this.cookieStore.get('kvu');
+		userHasEverLoggedInBefore(hasEverLoggedInBefore !== undefined);
+
+		const hasLentBeforeValue = this.cookieStore.get(hasLentBeforeCookie);
+		const hasDepositBeforeValue = this.cookieStore.get(hasDepositBeforeCookie);
+
+		if (hasLentBeforeValue === undefined || hasDepositBeforeValue === undefined) {
+			try {
+				let userData = {};
+				userData = this.apollo.readQuery({
+					query: optimizelyUserDataQuery,
+				});
+
+				const hasLentBefore = userData?.my?.loans?.totalCount > 0;
+				const hasDepositBefore = userData?.my?.transactions?.totalCount > 0;
+
+				this.cookieStore.set(hasLentBeforeCookie, hasLentBefore);
+				this.cookieStore.set(hasDepositBeforeCookie, hasDepositBefore);
```

Comment:
`userHasLentBefore` and `userHasDepositBefore` should be called here too.

Reply:
done!